lib/tasks/okc_blotter.rake

```diff
@@ -0,0 +1,10 @@
+namespace :okc_blotter do
+  desc 'Scape and parse'
+  task import_new: [:environment] do
+    Importers::OkcBlotter::Pdf.import_since_last_run
+  end
+
+  task import_missing: [:environment] do
+    Importers::OkcBlotter::Pdf.import_missing_dates
+  end
+end
```

Comment:
this is a Rake task which allows us to call ruby code from the command line

Reply:
got it, want to have quick access to your importers functionality.